cluster/ci/origin/images/build-base/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [[ -n "${UMASK}" ]]; then
+	umask "${UMASK}" && "$@"
```

Comment:
Sorry, I forget -- why did we need this indirection?

Reply:
Just to not have umask explicitly called wherever we run something... more of a hygiene thing.